eslint.config.js

```diff
@@ -5,9 +5,10 @@ import eslintPluginImport from 'eslint-plugin-import'
 import eslintPluginReact from 'eslint-plugin-react'
 // @ts-expect-error no types are available for this plugin
 import eslintPluginReactHooks from 'eslint-plugin-react-hooks'
+import eslintPluginVitest from '@vitest/eslint-plugin'
 
-const REACT_FILES_MATCH =
-  'packages/{tuono,tuono-router}/**/*.{js,mjs,cjs,jsx,mjsx,ts,tsx,mtsx}'
+const JS_EXTENSIONS_MATCH = 'js,mjs,cjs,jsx,mjsx,ts,tsx,mtsx'
+const REACT_FILES_MATCH = `packages/{tuono,tuono-router}/**/*.{${JS_EXTENSIONS_MATCH}}`
```

Comment:
What about making it enabled for all the packages? In case we need a new package, we risk forgetting about it

```suggestion
const REACT_FILES_MATCH = `packages/**/*.{${JS_EXTENSIONS_MATCH}}`
```

Reply:
I don’t understand why we should apply React rules to packages that aren't using React, like `tuono-fs-router-vite-plugin`. 
Running these rules on packages without `react` is a waste of resources and also issues a warning since the rules are unable to detect the react version to be used inside rules check.

Reply:
Yeah I agree. I don't have a better idea right now about preventing to miss updating this config. Let's go this way!